[Test] Add NGINX to k3d (#1336)
neicnordic/sensitive-data-archive: README.md

curl -s https://raw.githubusercontent.com/k3d-io/k3d/main/install.sh | bash
```

> #### Create a cluster

Reviewer: If the cluster is created by 
```
k3d cluster create test-cluster
```
The Nginx ingress controller will not be deployed. So maybe better to clarify that how one can create a cluster with a different name but still want to deploy the Nginx ingress controller. 

Or just remove it so that a new cluster should always be created by `make k3d-create-cluster`

kubectl get nodes
```

> The Nginx ingress controller is deployed and will bind to port 80 and 443 of the host system. As such a deployed service with an ingress definition can then be targeted by setting the `Host: HOSTNAME` header fo that service.

Reviewer: ```suggestion
The Nginx Ingress controller is deployed and will bind to ports 80 and 443 of the host system. As such, a deployed service with an Ingress definition can be targeted by setting the `Host: HOSTNAME` header for that service.
```

```sh
curl -H "Host: test" http://localhost/
```

> For testing ingress endpoints with other applications like a web browser, the hosts file `/etc/hosts` will ned to be edited unless some other form of DNS modifivcaion is used.

Reviewer: ```suggestion
For testing Ingress endpoints with other applications like a web browser, the hosts file `/etc/hosts` will need to be edited unless some other form of DNS modification is used.
```
